src/EditorFeatures/Core.Wpf/Interactive/ResetInteractive.cs

```diff
@@ -41,36 +41,35 @@ internal ResetInteractive(EditorOptionsService editorOptionsService, Func<string
             _createImport = createImport;
         }
 
-        internal Task ExecuteAsync(IInteractiveWindow interactiveWindow, string title)
+        internal async Task ExecuteAsync(IInteractiveWindow interactiveWindow, string title)
```

Comment:
caller of this was going an await anyways, so there was no reason i could figure out for this to be written as task returning, with task continuations, instead of just normal async/await.